app/models/user.rb

```diff
@@ -0,0 +1,6 @@
+class User < ApplicationRecord
+  # Include default devise modules. Others available are:
+  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
```

Comment:
- [ ] Please the user class should have the right association, the user has_many groups, and has_many items as in the diagram.
- [ ] There should be many to man relationships between the items and the groups.

![image](https://user-images.githubusercontent.com/74744787/219893085-cd210191-748f-40c6-84ba-3ea9c9254e51.png)